.travis.yml

```
@@ -10,6 +10,14 @@ branches:
   only:
     - master
 
+matrix:
+  fast_finish: true
+  include:
+    - env: NAME="Docs"
+           MAIN_CMD="cd docs && make html"
```

This works. However, I think we tend to use the OS-independent
```suggestion
           MAIN_CMD="python setup.py build_sphinx"
```
nowadays.

---

aha unfortunately #61. I am having serious trouble getting `build sphinx` to work. :(